App/components/StarsRating/starsRating.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ReactNode } from 'react'
+import { StyleSheet, Image, View, Text } from 'react-native'
+import * as Styled from './starsRating.styled'
+import SvgUri from 'react-native-svg-uri'
+
+type Props = {
+  children: ReactNode
+  score?: number
+  rating?: number
+}
+
+export const StarsRating = ({
+  children = '',
+  score = 10000,
+  rating = 3,
+}: Props) => {
+  let stars = []
```

Comment:
I think it would be more readable to have this code wrapped in a function like 
```javascript
const preparedStars = () => { .... return stars };
```